backend/LexBoxApi/GraphQL/UserMutations.cs

```diff
@@ -68,7 +70,10 @@ LexAuthService lexAuthService
 
         if (input is ChangeUserAccountByAdminInput adminInput)
         {
-            user.IsAdmin = adminInput.Role == UserRole.admin;
+            if (user.Id != loggedInContext.User.Id)
```

Comment:
I know this shouldn't be possible, but can we assert that the logged in user is an admin here.

Reply:
✅ https://github.com/sillsdev/languageforge-lexbox/commit/26fcd47368132e4cd28a99f7730b6b54eee461b8